packages/client/src/v2-events/features/events/actions/register/Review.tsx

```diff
@@ -146,8 +153,7 @@ export function Review() {
         transactionId: uuid(),
         annotation
       })
-
-      goToHome()
+      slug ? goToWorkqueue(slug) : goToHome()
```

Comment:
We could make a small helper functions instead of doing this same logic everywhere, something like:

```
  function goToWorkqueueOrHome(slug?: string) {
    if (slug) {
      goToWorkqueue(slug)
    } else {
      goToHome()
    }
  }
```